Tasks/PackerBuildV1/src/utilities.ts

```diff
@@ -107,7 +107,7 @@ export function findMatch(root: string, patterns: string[] | string): string[] {
 }
 
 export function getTempDirectory(): string {
-    return os.tmpdir();
+    return tl.getVariable('Agent.TempDirectory');
```

Comment:
Please update patch version in task.json

Reply:
Done